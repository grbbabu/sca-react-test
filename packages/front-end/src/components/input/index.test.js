import React from 'react';
import renderer from 'react-test-renderer';
import Input from './index';

it('renders correctly when there are no input to the form', () => {
    const tree = renderer
        .create(
            <Input
                label="First Name"
                type="text"
                id="firstName"
                name="firstName"
                value="firstName"
                onChange={e => e.target.value}
            />
        )
        .toJSON();
    expect(tree).toMatchSnapshot();
});
