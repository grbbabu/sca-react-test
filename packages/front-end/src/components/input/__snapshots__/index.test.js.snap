// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no input to the form 1`] = `
<div
  className={null}
>
  <label
    htmlFor="firstName"
  >
    First Name
    <input
      id="firstName"
      name="firstName"
      onChange={[Function]}
      required={false}
      rest={Object {}}
      type="text"
      value="firstName"
    />
  </label>
</div>
`;
